### ISSUE TITLE
New Awesome Unity list.

### ISSUE THREAD

**Issue by agarcialeon** [RyanNielson/awesome-unity#93]
Hi,

I have starred your repository long time ago and others lists with resources for Unity but I find that they aren't updated very often. For this reason I have created a new repository with the list of items included in each awesome unity lists that I have found (maybe some are still missing) and I have included all my starred repositories for Unity that I have saved during the past years.

I would like to see if could be possible to decide together how to group all the existent repositories into only one and improve it to make it a useful source of knowledge for Unity developers.

Could you give me your opinion, please? 
Thanks

Link to my repository:
[Awesome Unity](https://github.com/agarcialeon/awesome-unity)

**Comment by agarcialeon** [comment]
That's for sure. I'm not meaning people should promote their assets on these lists, because there are other channels for that. 

But even if it's curated there are plenty of assets out there that should be taken into account. 
Of course I could send you a PR with the resources I think you need to include but that won't make my suggestions have been reviewed by others to be later accepted.

I suggest to accept pull request for new contributions, but providing a template it.
This pull request should be reviewed at least by 2 other users in order to be accepted as with awesome list repository is driven or a similar way.

Issues could be used to add/modify/delete specific items or sections in order to agree on common groups of resources that everyone can find easily and if its not possible provide tools to search in other sources.

Also I have followed the guidelines for awesome lists and yours is missing some points and also mine but I think this point can be solved too.

Anyway my idea is bigger than just the curated list. I'm building a wiki in my repository for all type of Unity users and also planning to integrate github pages to create a blog that could be used as newsletter with updates of the repository.

If you think you think I could help your repository with anything else from my side or what I have collected, please tell me. But if you don't provide basic resources that Unity developers should know as a basis then a reorganization will be needed.

Don't take this as something bad against your repository. In fact I like it but I think we can make a better list (yes, curated) and I would like to see users have these kind of lists as a reference.

**Comment by agarcialeon** [comment]
I'm going to add other repository administrators to the thread to get more feedback I you don't mind.

These for the moment (based on stars and first ocurrences):

- @netpyoung from Awesome-unity-free.
- @QianMo from Awesome-Unity-Shader.
- @insthync from awesome-unity3d.
- @darkwind666 from awesome-unity-plugins-assets-tools-services-games-projects-resources.
- @pixelfoot from unity-awesomeness.
- @UnityStudentAmbassadors from UnityStudentAmbassadors.
- @ocularrhythm from unity-awesome.

I will contact more administrators later.

**Comment by ocularrhythm** [comment]
I created my list much in the same spirit as @RyanNielson, such that intended mine to be more of a curated list that I _personally_ found valuable or interesting. For that reason I had originally named my repository "Unity Radical," as I didn't want it to be confused as an "awesome" list (unofficially, I also irrationally hate the word "awesome").

My list doesn't get updated often just because my interests constantly shift, but that's irrelevant. I think your concerns are perfectly valid @agarcialeon and it seems like the proposal you are presenting seems like a good idea. I certainly don't want to speak for Ryan as this is his repository though so that is a discussion reserved between you two.

I'm not entirely sure what you need from me, but I would be happy to add this and any other awesome list repos to mine as well so at the very least they can be somewhat interconnected. Or if fragmentation of lists is what you're chiefly concerned with, I could also either rename mine back to something like "radical" to reduce confusion, or mark mine private all together.

**Comment by agarcialeon** [comment]
Well, in that case, what I could do is rename my repository to another name to avoid duplication with @RyanNielson but create a section that links to all other repositories as yours @ocularrhythm .

What I wanted from other administrator is feedback about how to proceed when creating the repository I'm building because even if should be curated I wanted to gather other resources from other repositories. 

However I've seen there are some of them with the same goals as mine and instead of creating another list that people could forget, we could agree on something common and redirect users to each other repository or maybe join all the resources in only one.

So, in your honest opinion, what can I do with my repository to help you and other users when creating a list of resources for unity? Would you like to move my resources to another repository as @RyanNielson's? Or have another one with a different name that references the others?

And you are correct about fragmentation. That's one of my concerns, since I find that Unity already has a lot of scattered sources where to learn.

**Comment by ocularrhythm** [comment]
tl;dr - I say move forward with your own list and we can just link to each others

Well, I think that regardless of what agreement we came to moving forward with the lists, you will constantly run the risk of someone else creating an additional list with no knowledge of this conversation taking place. Honestly, I think the best thing for you to do is to just create your resource as you envision and if it turns out to be the "best" list out there, it will naturally be elevated through stars, or pull requests, or forks, etc.

The fragmentation is going to happen and there's not really much you could do to stop that. I was already aware of the other lists when I created my own, but I did it because I wanted a smaller list. I say that because everyone's motivations could be different. You want to create a "master" all-encompassing list that helps both beginners and newbies alike. Whereas I want my list to be deliberate and specific to my journey.

**Comment by agarcialeon** [comment]
Ok. Thanks for the feedback. Then I'll try that approach and reference other lists. I'll try to update the repository name in order to not conflict with @RyanNielson.
